dataland-frontend/src/components/generics/ViewFrameworkBase.vue

```diff
@@ -129,12 +153,12 @@ export default defineComponent({
     return {
       getKeycloakPromise: inject<() => Promise<Keycloak>>("getKeycloakPromise"),
       frameworkDataSearchBar: ref<typeof FrameworkDataSearchBar>(),
+      hideEmptyFields: ref(true),
```

Comment:
Is there a reason why this is in the setup block and not in data as usual? Also, why is it a ref?

Reply:
Ok, I tested that out and it is not necessary to define this in setup and as a ref as well. If there is no good reason against it lets move it into the data block without ref.

Reply:
It is now part of data{}